docs-src/typescript/how-to-import-an-ecmascript-module.md

```diff
@@ -0,0 +1,16 @@
+---
+id: how-to-import-an-ecmascript-module
+title: How to import an ECMAScript module
+sidebar_label: ECMAScript modules
+description: See our Fetch ESM sample for the necessary configuration changes.
+tags:
+  - guide-context
+---
+
+The JavaScript ecosystem is increasingly moving toward publishing ECMAScript modules (ESM) instead of CommonJS modules; for example, `node-fetch@3` is ESM, but `node-fetch@2` is CommonJS.
```

Comment:
```suggestion
The JavaScript ecosystem is quickly moving toward publishing ECMAScript modules (ESM) instead of CommonJS modules.
For example, `node-fetch@3` is ESM, but `node-fetch@2` is CommonJS.
```

Reply:
Also, I would add a mention about what this means for updates. Like, keep an eye on this ecosystem change because it can affect code that uses these modules, something like that.